data/sql/updates/pending_db_world/rev_1648131745456322279.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1648131745456322279');
+
+-- for Zone Orgrimmar
+REPLACE INTO `gossip_menu` (`MenuID`, `TextID`) VALUES
```

Comment:
Please Use Insert Instead of replace

Reply:
> Please Use Insert Instead of replace

Thanks for replying!

But... What for? For complicate the syntax and double the SQL-query? Usage of REPLACE is the best variant instead DELETE + INSERT
I never use it and don't recommend it to you. This synthax is a complete replacement for the double syntax.
If you want to control what wil be changed - you can see it all in (???, ???). So, don't need to delete it befor inserting. Just use REPLACE and be happy :)

Reply:
Because it is what AzerothCore has decided, see https://www.azerothcore.org/wiki/sql-standards#insert-amp-delete